test/kubemark/resources/hollow-node_template.json

```diff
@@ -145,27 +157,48 @@
 				},
 				{
 					"name": "hollow-node-problem-detector",
-					"image": "gcr.io/google_containers/node-problem-detector:v0.2",
+					"image": "gcr.io/google_containers/node-problem-detector:v0.3",
```

Comment:
@wojtek-t Thanks for pointing this.
@Random-Liu I assume this is a bug (and we haven't really bothered about this as we haven't yet released v0.3 officially). The entrypoint in the Dockerfile (https://github.com/kubernetes/node-problem-detector/blob/master/Dockerfile) should be removed as we need to trigger the binary through a command from now on (both in standalone or DaemonSet mode).